lib/smartsheet/rows.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Smartsheet.Rows do
+  def add_to_sheet(sheet_id, row = %{}) do
+    Smartsheet.Rows.add_to_sheet(sheet_id, [row])
+  end
+
+  def add_to_sheet(sheet_id, rows) do
+    path = "/sheets/#{sheet_id}/rows"
+
+    {:ok, response = %HTTPoison.Response{}} =
+      Smartsheet.client().post(path, rows, "Content-Type": "application/json")
```

Comment:
Does the content type ever vary? Seems like the client just set that top level rather than each function having to set it.

Reply:
It's only necessary for POSTS and PUTS. I tried to override this at the client level but couldn't figure out a way to shim the HTTPoison method to conditionally include headers based one the action. /shrug i mean we could send it all the time, it's just not technically "correct"

Reply:
Ahh it's because you're calling the HTTPoison get here, yeah you're right I think this is the right thing for now.